pkg/cni/install_test.go

```diff
@@ -44,6 +44,7 @@ func TestWatchTokenFile(t *testing.T) {
 	cniDir := t.TempDir()
 
 	i := NewInstaller(constants.WorkloadMode, cniDir, "conflist-name", true, serviceAccountPath)
+	defer i.Watcher.Close()
```

Comment:
should use i.Stop

Reply:
I didn't call `i.Start()`, so the entire cni installation process was not simulated, so calling `i.Stop()` would fail actually.